BioForm/profiles/views.py

```diff
@@ -62,6 +61,37 @@ def logout_user(request):
     logout(request)
     return redirect("login")
 
+@login_required(login_url = 'login')
+def display_bio(request):
+    all_user_data = Bio.objects.filter(user_id = request.user)
```

Comment:
Just `user` is enough adding `all` gives the impression it may have data of more than one user.